bundles/io/org.openhab.io.caldav/src/main/java/org/openhab/io/caldav/internal/job/EventReloaderJob.java

```diff
@@ -122,24 +123,17 @@ public void execute(JobExecutionContext context) throws JobExecutionException {
 
             // printAllEvents();
         } catch (SardineException e) {
-            log.error(
-                    "error while loading calendar entries: " + e.getMessage() + " (" + e.getStatusCode() + " - " + e.getResponsePhrase() + ")",
-                    e);
-            throw new JobExecutionException(
-                    "error while loading calendar entries", e, false);
+            log.error("error while loading calendar entries: " + e.getMessage() + " (" + e.getStatusCode() + " - "
```

Comment:
While I agree with you @9037568 Chris that parameterized logging is always preferable, it matters less on error paths, and if this code was only changed because it was previously formatted incorrectly, it would be reasonable to leave it as is.  But it's nevertheless not a bad opportunity to make it cleaner if @aschor doesn't mind.  The new line would thus read:

```
log.error("error while loading calendar entries: {} ({} - {})", e.getMessage(), e.getStatusCode(), e.getResponsePhrase(), e);
```
BUT...looking at [this issue](https://github.com/lookfirst/sardine/issues/218), if Sardine is ever upgraded to 5.7, then a simple `getMessage()` would only need to be logged.